### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by chaitanyapotti**
## Motivation and Context

Cleanup unused parameters
Jira Link:

## Description

Cleanup unused parameters

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project. (run lint)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] My code requires a db migration.

### CODE REVIEW

File: Sources/Web3Auth/Web3Auth.swift [hunk 1]
```diff
@@ -153,11 +153,6 @@ public class Web3Auth: NSObject {
      - parameter callback: Callback called with the result of the WebAuth flow.
      */
     public func login(_ loginParams: W3ALoginParams) async throws -> Web3AuthState {
-        // Think we should avoid this and throw error instead
```
Comment (metalurgical) [inline Sources/Web3Auth/Web3Auth.swift:156]: Perhaps this instead? Similar for the other one.

```
if initParams.redirectUrl == nil {
   // throw error here
}
```

Comment (metalurgical) [reply Sources/Web3Auth/Web3Auth.swift:156]: Since bundleid was originally being used to ensure redirecturl was not nil.

**Review by grvgoel81 — APPROVED** [review]
lgtm.

File: Sources/Web3Auth/Web3Auth.swift (hunk 1 above)
Comment (chaitanyapotti) [reply Sources/Web3Auth/Web3Auth.swift:156]: added the checks now

**Review by metalurgical — APPROVED** [review]

**Pull request merged**